tsung/templates/tsung-form-submission.xml.j2

```diff
@@ -47,8 +57,11 @@
 
 
             &save_form;
+            <thinktime value="2" random="true"></thinktime>
```

Comment:
actually the reason the thinktimes are high in the examples is because they're simulating users thinking about clicking the next button. in our case we're using thinktime as the time the code takes to process